test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -930,6 +930,467 @@ test.describe( 'Navigation block', () => {
 			await expect( linkControl.getSearchInput() ).not.toBeVisible();
 		} );
 	} );
+
+	test.describe( 'Navigation colors', () => {
+		test.beforeAll( async ( { requestUtils } ) => {
+			//we want emptytheme because it doesn't have any styles
+			await requestUtils.activateTheme( 'emptytheme' );
+			await requestUtils.deleteAllTemplates( 'wp_template_part' );
+			await requestUtils.deleteAllTemplates( 'wp_template' );
+			await requestUtils.deleteAllPages();
+			await requestUtils.deleteAllMenus();
```

Comment:
Why do we need to do this bit?

Reply:
We are editing the theme every time we run the tests by creating a new menu and inserting in the header template part, this is how we reset it. I changed how we do this a few times so we probably don't need to delete the pages and the templates, but we do need to keep removing template parts and menus. I'll double check this.